<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diem_monhoc</class>
 <widget class="QWidget" name="diem_monhoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_topic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> BẢNG ĐIỂM MÔN</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>811</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="BangDiem_Widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>791</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TraBangDiem">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tra Bảng Điểm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nienkhoa">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Niên Khóa : 0000-0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hocky">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Học kỳ : _</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nhom">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nhóm : _</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>561</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Niên khóa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit__TenMon">
      <property name="placeholderText">
       <string>Chữ cái</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit__Hocky">
      <property name="placeholderText">
       <string>Số thường</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Học kỳ</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit__NienKhoa">
      <property name="placeholderText">
       <string>xxxx-xxxx</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Nhóm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit__Nhom">
      <property name="placeholderText">
       <string>Số Thường</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Tên môn</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableView</zorder>
  <zorder>label_topic</zorder>
  <zorder>BangDiem_Widget</zorder>
  <zorder>pushButton_TraBangDiem</zorder>
  <zorder>label_nienkhoa</zorder>
  <zorder>label_hocky</zorder>
  <zorder>label_nhom</zorder>
  <zorder>gridLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
